src/value_converter.rs

```diff
@@ -133,6 +134,37 @@ impl PythonDTO {
             )),
         }
     }
+
+    /// Get value from DTO
+    ///
+    /// # Errors
+    /// May return Err Result if cannot convert python type into rust.
+    pub fn get_value(&self) -> RustPSQLDriverPyResult<Value> {
```

Comment:
What is the difference from `to_serde_value` method?

Reply:
Looks like code duplicate

Reply:
I added this to show what i need, removed it for now